docker-compose.yml

```diff
@@ -64,6 +64,7 @@ services:
       - NETNS_MODE=off
       - REDIS_QUEUE_HOST=redis-queue
       - REDIS_STATE_HOST=redis-state
+      - OVERRIDE_CPU_COUNT=4
```

Comment:
Setting a lower CPU count on each worker forces function executions to be distributed. Can be adjusted for smaller hosts too.